CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Mapbox welcomes participation and contributions from everyone.
 #### Bug fixes and improvements
 - Optimized calls to modify route arrow layer visibility. [#6308](https://github.com/mapbox/mapbox-navigation-android/pull/6308)
 - Provide better java support for `MapboxNavigationApp`. [#6292](https://github.com/mapbox/mapbox-navigation-android/pull/6292)
+- Added a try/catch and logging around calls to get string resources related to distance formatting suffixes. [#6309](https://github.com/mapbox/mapbox-navigation-android/pull/6309)
```

Comment:
Is this really supposed to be in CHANGELOG? Does it affect users in any way? If there were related crashes, maybe we should just specify which ones are fixed by this?